pcli/src/network.rs

```diff
@@ -55,6 +63,20 @@ impl Opt {
 
         if code == 0 {
             println!("transaction submitted successfully");
+
+            if let Some(note_commitment) = await_change {
+                let mut view_client = self.view_client(fvk, self.data_path.as_ref()).await?;
```

Comment:
We don't want to do this, because with the current `view_client` implementation, this will potentially spin up a new view service for every submitted transaction. (If `pcli` is using an external `pviewd`, it will just create a new connection, but even that is suboptimal relative to reusing it).